src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -2594,6 +2610,13 @@ module Queries = struct
         | None ->
             [] )
 
+  let protocol_amounts =
+    field "protocolAmounts"
+      ~doc:"The currency amounts for different events in the protocol"
```

Comment:
Doc: "The constants used to determine the configuration of the genesis block and all of its transitive dependencies" (something like that)